src/test/kotlin/com/google/firebase/analytics/FirebaseAnalytics.kt

```diff
@@ -1,14 +1,17 @@
 package com.google.firebase.analytics
 
-import java.util.LinkedList
-import java.util.AbstractMap.SimpleEntry
 import android.app.Activity
 import android.content.Context
 import android.os.Bundle
+import java.util.AbstractMap.SimpleEntry
+import java.util.LinkedList
 
 class FirebaseAnalytics {
     var loggedEvents: LinkedList<Map.Entry<String, Bundle>> = LinkedList()
     var currentScreenName: String? = null
+    var consentStateMap:MutableMap<Any, Any> = mutableMapOf()
+
+
```

Comment:
Nitpick about spacing. Looks like these two added lines now make 3 lines of whitespace, maybe remove lines 13 and 14?

I tried to do it with a change suggestion but github wasn't parsing it correctly.